xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2494,6 +2494,12 @@ bool CActiveAE::HasDSP()
   return m_stats.HasDSP();
 };
 
+bool CActiveAE::GetCurrentSinkFormat(AEAudioFormat &SinkFormat)
+{
+  SinkFormat = m_sinkFormat;
```

Comment:
m_sinkFormat is not supposed to be accessed by any thread other than AE's thread